src/main/resources/templates/choreDetails.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https//www.thymeleaf.org/">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Title</title>
+</head>
+<body class="container">
+
+<table class="table table-striped">
+    <tr>
+        <th>Chore</th>
+        <td th:text="${chore.id}"></td>
+    </tr>
+    <tr>
+        <th>Description</th>
+        <td th:text="${chore.choreDetails.description}"></td>
```

Comment:
```suggestion
        <td th:text="${chore.description}"></td>
```

Reply:
Right idea; we currently don't have a `ChoreDetails` object like we had created in Coding Events (where we made `EventDetails`). You could make one, though, if you think it will clear things up!